CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [3.7]
+
+- **Change** Added new star icon with more rounded edges (`roudned-star`)
```

Comment:
`roudned-star` - typo! rounded is spelled wrong but it looks like this should be `star-rounded` anyway

Reply:
`star--rounded` does seem correct

Reply:
The sketch file is called `star-rounded`. I don't know why after running `yarn run build` it created an icon called `rounded-star`.

Reply:
@osermay Then I guess we will have `star--rounded--small` for small variant.

Reply:
Okay. I get this. I've downloaded and installed Sketch and renamed the artboards inside Sketch app.